deploy/operator/operator.yaml

```diff
@@ -74,6 +74,8 @@ spec:
               value: ""
             - name: HANDLER_IMAGE
               value: {{ .HandlerImage }}
+            - name: PLUGIN_IMAGE
+              value: quay.io/nmstate/nmstate-console-plugin:release-1.0.0
```

Comment:
I don't think we should have references to upstream images in the downstream repo, even if we expect it to be replaced by ART. Can we point this at the openshift image?

Reply:
updated the image pullspec to:
`registry.redhat.io/openshift4/nmstate-console-plugin-rhel8:v4.13`
as appeared in the delivery repo:
https://comet.engineering.redhat.com/containers/repositories/63dd4b03ada9b035ee004067?tab=configuration

Reply:
Hi @cybertron ,
I tried specifying the downstream image in `registry.redhat.io`, but the test cluster can't pull it since the D/S console-plugin image hasn't been published yet.

so I tried to use instead: `brew.registry.redhat.io/rh-osbs/openshift-nmstate-console-plugin-rhel8:v4.13` which is available externally, but requires a pull-secret for brew.registry.redhat.io, which apparently the prow job lacks this pull-secret.

I see the other images are referencing to `quay.io/openshift/origin-kubernetes-nmstate-handler:4.13` and `quay.io/openshift/origin-kubernetes-nmstate-operator:4.13`.
How these images are being pushed to quay.io?
Is it possible to push also the console plugin image to `quay.io/openshift/origin-kubernetes-nmstate-console-plugin:4.13` ?

Thanks